sdk/src/manifest.rs

```diff
@@ -1023,11 +1023,30 @@ impl Manifest {
 
     /// Embed a signed manifest into a stream using a supplied signer.
     /// returns the bytes of the new asset
+    /// We plan to deprecate this, please use embed_to_stream instead
```

Comment:
Should we formally mark this as deprecated?

https://doc.rust-lang.org/reference/attributes/diagnostics.html#:~:text=The%20deprecated%20attribute&text=rustdoc%20will%20show%20item%20deprecation,string%20in%20the%20deprecation%20message.